pages/guide.py

```diff
@@ -241,7 +241,7 @@ def process_post_data(self, **kwargs):
     if self.touched('ready_for_trial_url'):
       feature.ready_for_trial_url = self.parse_link(
           'ready_for_trial_url')
-      update_items.append(('ready_for_trial_url',
+      stage_update_items.append(('ready_for_trial_url',
           self.parse_link('ready_for_trial_url')))
```

Comment:
```suggestion
      stage_update_items.append(('announcement_url',
          self.parse_link('ready_for_trial_url')))
```
The first item of this tuple represents the field that should be changed, and the second item is the value. This change can be made and the `elif` statement also added on this file can be removed.

Reply:
Actually, this is a bit more complex, and you'll need to add a key value pair to `core_enums.STAGE_TYPES_BY_FIELD_MAPPING` to ensure this is saved to the right stage as well.

```
'announcement_url': STAGE_TYPES_DEV_TRIAL,
```

Reply:
Good point. Changed.